site/content/docs/main/file-system-backup.md

```
@@ -555,33 +546,21 @@
 
 - `BackupRepository` - represents/manages the lifecycle of Velero's backup repositories. Velero creates 
 a backup repository per namespace when the first FSB backup/restore for a namespace is requested. The backup 
-repository is backed by restic or kopia, the `BackupRepository` controller invokes restic or kopia internally, 
-refer to [restic integration](#how-velero-integrates-with-restic) and [kopia integration](#how-velero-integrates-with-kopia) 
-for details.
+repository is backed by kopia, the `BackupRepository` controller invokes kopia internally, 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)